app/data/test_total_breakdown.json

```diff
@@ -0,0 +1,131 @@
+                                            "validation": {
+                                                "messages" : {
+                                                    "INTEGER_TOO_LARGE": "The maximum value allowed is 100. Please correct your answer."
+                                                }
+                                            }
+                                        },
+                                        {
+                                            "description": "",
+                                            "id": "percentage-3",
+                                            "label": "Percentage 3",
+                                            "mandatory": false,
+                                            "options": [],
+                                            "q_code": "0830",
+                                            "type": "Percentage",
+                                            "validation": {
+                                                "messages" : {
+                                                    "INTEGER_TOO_LARGE": "The maximum value allowed is 100. Please correct your answer."
+                                                }
+                                            }
+                                        },
+                                        {
+                                            "description": "",
+                                            "id": "percentage-4",
+                                            "label": "Percentage 4",
+                                            "mandatory": false,
+                                            "options": [],
+                                            "q_code": "0840",
+                                            "type": "Percentage",
+                                            "validation": {
+                                                "messages" : {
+                                                    "INTEGER_TOO_LARGE": "The maximum value allowed is 100. Please correct your answer."
+                                                }
+                                            }
+                                        },
+                                        {
+                                            "calculated": true,
+                                            "calc_method": {
+                                                "name": "sum",
+                                                "values": [
+                                                    "percentage-1",
+                                                    "percentage-2",
+                                                    "percentage-3",
+                                                    "percentage-4"
+                                                ]
+                                            },
+                                            "description": "<strong>The total percentages should be 100%</strong>",
```

Comment:
Not sure we should be using `description` for error messages. This is a new feature that needs thinking about carefully as it has accessibility implications.